### PR TITLE
feat(nightwatch): Add support for Nightwatch v1.x

### PR DESCRIPTION
**Pull request by ruben0626**
- Add support for Nightwatch v1.x

### CODE REVIEW

File: src/polyfill/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    if (this.message === undefined) {
+      this.message = "";
+    }
+
+    if (this.expected === undefined) {
+      this.expected = "";
+    }
+
+    if (this.pass === undefined) {
```
Comment (zzo) [inline src/polyfill/index.js]: typeof this.pass != 'function'  and for this.value ?

File: src/polyfill/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  }
+}
+
+/* eslint-disable  no-invalid-this,prefer-spread */
+export const exportLegacyAssertions = function(LegacyAssertion, module) {
+  const assertion = function() {
+    const args = arguments;
+    const instance = new LegacyAssertion(this.client);
+
+    if (this.message === undefined) {
```
Comment (zzo) [inline src/polyfill/index.js]: why not just !this.message and !this.expected ?

File: src/polyfill/index.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    if (this.pass === undefined) {
+      this.pass = () => false;
+    }
+
+    if (this.value === undefined) {
+      this.value = (value) => value;
+    }
+
+    this.command = function(callback) {
+      const output = instance.command.apply(instance, args);
```
Comment (zzo) [inline src/polyfill/index.js]: just `instance.command(...args)` ?

File: tests/src/polyfill/index.test.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+const expect = chai.expect;
+
+describe("Nightwatch Polyfill", () => {
+  it('should add legacy properties from NW v0.x that are used by NW-extra', () => {
+    const nightwatchClient = {
+      settings: {
+        desiredCapabilities: {},
+      },
+    };
+    expect(nightwatchClient.assertion).to.be.undefined;
```
Comment (zzo) [inline tests/src/polyfill/index.test.js]: can any of these assertions be more specific other than just 'not defined' ?

File: src/polyfill/index.js (hunk 2 above)
Comment (ruben0626) [reply src/polyfill/index.js]: Because `message = null` and `expected = false` are valid values.

File: src/polyfill/index.js (hunk 3 above)
Comment (ruben0626) [reply src/polyfill/index.js]: For some reason this didn't work

**Pull request merged**